plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
 
 **Possible Values:** `true`, `false`
 
-#### Example: a11ychecker_allow_decorative_images
+#### Example: Using `a11ychecker_allow_decorative_images`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_allow_decorative_images: true
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_allow_decorative_images: true
+});
+```
+
+### `a11ychecker_filter_issue`
+
+The `a11ychecker_filter_issue` option allows Accessibility Checker rules to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `a11ychecker_ignored_rules` option.
+
+The function will be passed the result object of the [`getReport()` API](#getreport). To filter issues from the Accessibility Checker report, the callback needs to return `false` for the issue.
+
+**Type:** `Function`
+
+**Default Value:**
+
+```js
+function(issue) {
+  return true;
+}
+```
+
+#### Example: Using `a11ychecker_filter_issue` filter the Accessibility Checker T1 rule
+
+The callback function in the following example will return `false` if the issue `id` value is `'T1'`, filtering `'T1'` issues from the Accessibility Checker report.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_filter_issue: function(issue) {
+    return issue.id !== 'T1';
+  }
+});
+```
+
+#### Example: Using `a11ychecker_filter_issue` filter all Accessibility Checker table rules
+
+The callback function in the following example will return `false` if the issue `element` value is `'table'`, filtering all table-related issues from the Accessibility Checker report.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_filter_issue: function(issue) {
+    return issue.element.nodeName.toLowerCase() !== 'table';
+  }
 });
 ```
 
 ### `a11ychecker_html_version`
 
 This configuration option sets the HTML version to use when checking issues.
 
-For example, setting the version to HTML 4 will trigger the rule "Complex tables should have summaries" as summary is a valid attribute and is required for tables, however in HTML 5 the attribute is deprecated, so the rule won't be triggered.
+For example: Setting the version to HTML 4 will trigger the rule "Complex tables should have summaries", where `summary` is a valid attribute in HTML 4 but not HTML 5.
```

Comment:
I kind of prefer the level of detail in the old explanation. However, either way we should probably change it from "will trigger the rule" to some variation of either "will allow the rule to be triggered" or "will trigger the rule on [example HTML]"

Reply:
"will allow the rule to be triggered" is a passive-voice version of the current change, which we do not want ;)
to improve clarity, I'll replace "trigger" with "enable"

Reply:
```suggestion
For example: Setting the version to HTML 4 will enable the rule "Complex tables should have summaries", where `summary` is a valid attribute in HTML 4 but not HTML 5.
```